spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -108,7 +112,9 @@ def send_login_request(user, password)
     end
 
     # rubocop:disable Metrics/PerceivedComplexity
-    def handle_itc_response(raw)
+    # If the response is coming from a flaky api, set flaky_api_call to true so we retry a little.
```

Comment:
yes, we totally should, I just wanted to be surgical about it. I wasn't 100% sure which apis needed this yet.